scripts/md2html/build.sh

```diff
@@ -10,17 +10,28 @@ mkdir -p deploy/js
 
 cd scripts/md2html
 mkdir -p history
-git show c740e950d:MAINTAINERS.md > history/MAINTAINERS_v2.0.md
+cat > history/MAINTAINERS_v2.0.md <<EOF
+## Active
+* Jeremy Whitlock [@whitlockjc](https://github.com/whitlockjc)
+* Marsh Gardiner [@earth2marsh](https://github.com/earth2marsh)
```

Comment:
@whitlockjc @earth2marsh please confirm, my OpenAPI history knowledge is spotty.

Reply:
@webron remains an active maintainer, too! otherwise lgtm (assuming I understood the question)

Reply:
Yes, @webron is an active maintainer in all versions of `MAINTAINERS.md`.

The question is whether I listed all active editors of version 2.0, and whether there were any changes between publication of 2.0 and 3.0.0.